crates/blockifier/src/execution/native_syscall_handler.rs

```diff
@@ -147,16 +147,16 @@ impl<'state> StarkNetSyscallHandler for NativeSyscallHandler<'state> {
         // Get Transaction Info
         let tx_info = &self.execution_context.tx_context.tx_info;
         let mut native_tx_info = TxV2Info {
-            version: starkfelt_to_felt(tx_info.signed_version().0),
+            version: native_stark_felt_to_felt(tx_info.signed_version().0),
```

Comment:
native_stark_felt_to_felt doesn't make too much sense, shouldn't it be "native_felt_to_stark_felt"?

Reply:
ah, I see, I misunderstood you a little